parley/src/context.rs

```diff
@@ -20,7 +20,7 @@ use swash::text::cluster::CharInfo;
 use crate::builder::TreeBuilder;
 use crate::inline_box::InlineBox;
 
-/// Context for building a text layout.
```

Comment:
It's not clear to me why you've removed the full stops from these comments. Our docs style (matching `std`) is to have a full stop at the end of the first sentence.

Reply:
No particular reason. Just wasn't paying attention to them.